### PR TITLE
Pagination Widget

### PR DESCRIPTION
**Pull request by cjohannsen**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested by the author
- [ ] Changes have been tested by the reviewer
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/lang/de/Widget.properties [hunk 1]
```diff
@@ -847,3 +847,8 @@ ItemsPerPageEntryNameTooltip = "Anzahl der angezeigten Artikel eingeben. Kunden
 toolbarLabel = "Toolbar"
 toolbarTextLabel = "Text"
 toolbarTextTooltip = "Beschriftung des Buttons auswählen."
+; Pagination
+paginationLabel = "Seitennummerierung"
+paginationPageLimitLabel = "Zus. Seiten pro Richtung beschränken"
+paginationShowFirstPageLabel = "Erste Seite immer anzeigen"
```
Comment (cjohannsen) [inline resources/lang/de/Widget.properties:853]: @fmutschler wir haben darüber gesprochen

File: resources/lang/en/Widget.properties [hunk 2]
```diff
@@ -846,3 +846,8 @@ ItemsPerPageEntryNameTooltip = "Enter the number of displayed items per page. Cu
 toolbarLabel = "Tool bar"
 linkTextLabel = "Button label"
 linkTextTooltip = "Enter a label for the button."
+; Pagination
+paginationLabel = "Pagination"
+paginationPageLimitLabel = "Restrict additional pages in each direction"
+paginationShowFirstPageLabel = "Always show first page"
```
Comment (cjohannsen) [inline resources/lang/en/Widget.properties:852]: @fmutschler wir haben darüber gesprochen

File: resources/scss/ceres/widgets/Helper/_widget-appearance.scss [hunk 3]
```diff
@@ -58,20 +58,25 @@
...
-                    color: darken($value, 15%);
-                }
-            }
+            flex-wrap: wrap;
 
-            &.active .page-link
+            .page-item
             {
-                border-color: $value;
-                @include context-bg($value);
```
Comment (Tim-M-S) [inline resources/scss/ceres/widgets/Helper/_widget-appearance.scss:74]: Hier fehlt noch cursor: pointer

Comment (cjohannsen) [reply resources/scss/ceres/widgets/Helper/_widget-appearance.scss:74]: passt nicht in diesem Kontext

**Review by Tim-M-S — APPROVED** [review]

**Comment by maxhelligeplenty** [comment]
Hallo  ,
Tim-M-S approved this pull request.





-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/plugin-ceres/pull/1598#pullrequestreview-310990015

Test.test.test

**Comment by maxhelligeplenty** [comment]
Hallo  ,
Merged #1598 into feature/beta/sb_categoryitem.

-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/plugin-ceres/pull/1598#event-2767547904

Test.test.test

**Pull request merged**